src/components/new-safe/steps/Step2/index.tsx

```diff
@@ -86,27 +93,7 @@ const CreateSafeStep2 = ({
                 </span>
               </Tooltip>
             </Typography>
-            <Typography variant="body2">
-              Add an extra layer of security and sign transactions with the Safe Mobile app.
-            </Typography>
-          </Grid>
-
-          <Grid item xs={12}>
-            {mobileOwnerFields.map((field, i) => (
-              <OwnerRow
-                key={field.id}
-                groupName={CreateSafeStep2Fields.mobileOwners}
-                index={i}
-                remove={removeMobileOwner}
-              />
-            ))}
-            <Button
-              variant="text"
-              onClick={() => appendMobileOwner({ name: '', address: '' }, { shouldFocus: true })}
-              startIcon={<SvgIcon component={AddIcon} inheritViewBox fontSize="small" />}
-            >
-              Add mobile owner
-            </Button>
+            <Typography variant="body2">Use your mobile phone as your additional owner key</Typography>
```

Comment:
This now seems oddly placed. I would say it's more of a hint now. What do you think? cc @liliiaorlenko 

![image](https://user-images.githubusercontent.com/20442784/199453129-540b5472-4a36-40fb-8ff3-91bb244fc808.png)

Reply:
https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=56%3A6575
how about adding the bg for the hint to have more separation?